### PR TITLE
magento/devdocs#: Cleanup. legacy-file-system-perms.md

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) makes a cleanup regarding to notice:

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/install-gde/install-quick-ref.html
- https://devdocs.magento.com/guides/v2.3/install-gde/install/file-sys-perms-over.html
- https://devdocs.magento.com/guides/v2.3/install-gde/install/prepare-install.html
- https://devdocs.magento.com/guides/v2.3/install-gde/prereq/file-sys-perms-over.html
- https://devdocs.magento.com/guides/v2.3/install-gde/prereq/file-system-perms.html
- https://devdocs.magento.com/guides/v2.3/install-gde/prereq/zip_install.html

## Links to Magento source code

-  ...

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by keharper** [comment]
@atwixfirster There's a merge conflict and I'm not sure where it's coming from.

**Comment by atwixfirster** [comment]
> @atwixfirster There's a merge conflict and I'm not sure where it's coming from.

I've renamed src/_includes/install/file-system-perms-before.md to src/_includes/install/file-system-perms-before_23.md in my PR.

Working on this problem now.

**Comment by atwixfirster** [comment]
@keharper , I've solved a merge conflict. Thanks

**Comment by shrielenee** [comment]
@keharper Hey, what is up with this one? Thanks! 🏆

File: src/_data/toc/installation-guide.yml [hunk 1]
```diff
@@ -285,6 +285,7 @@ pages:
 
         - label: Appendix—Magento file system ownership and appendix (legacy)
           url: /install-gde/install/legacy-file-system-perms.html
+          include_versions: ["2.2"]
```
Comment (dobooth) [inline src/_data/toc/installation-guide.yml]: Should this be `exclude_versions`?

**Comment by keharper** [comment]
Closing this PR because it was opened when there was a v2.2 directory. The PR renamed several files to differentiate between 2.2 and 2.3 behavior, and that's no longer necessary. I will approve a new, simpler PR.

**Pull request closed without merging**